apps/scan/backend/src/store.ts

```diff
@@ -545,64 +505,6 @@ export class Store {
     return row?.ballotsCounted ?? 0;
   }
 
-  /**
-   * Returns whether the appropriate backups have been completed and it is safe
-   * to unconfigure a machine / reset the election session. Always returns
-   * true in test mode.
-   */
-  getCanUnconfigure(): boolean {
```

Comment:
OK, so now that we have continuous backup you can always reconfigure because it's always backed up. Yes?

When the USB is out of sync and needs to be updated it shows a warning and you click a button to sync. But IIRC it shows the prompt via the voter screen, and the election manager screens are still accessible normally with authentication. So does this mean that you could have the "out-of-sync" warning, insert an Election Manager card, and unconfigure anyway? I haven't confirmed this is actually possible so may just be missing something.

Reply:
This line of reasoning is all correct - it _is_ possible to access the PW, EM, and SA screens when the sync prompt is up on the voter screen.

I'd been thinking about how worth it it was to prevent unconfiguring while in this fairly rare state. Things are complicated by the fact that the USB likely won't be in the machine come time to unconfigure, since it'll have been taken to VxAdmin for tabulation. And we don't display the sync prompt when there's no USB in the machine at all. Which brings us back to the need for some separate piece of store state if we want to fully track this.

I'm leaning toward disabling the PW polls-close button and the EM delete-election-data button when in the sync-required state, as the simplest possible safeguard.

Yes, an EM could still remove the USB and then delete election data (since the EM screen is still accessible when there's no USB). But a PW couldn't remove the USB and then close polls (since the PW screen is not accessible when there's no USB). So if a machine has had its polls closed, it's guaranteed to be backed up.

I think that this covers our bases well but can double check with Roe, as well!

Reply:
And got confirmation that we're good to go with the edits that I've proposed in this thread ✅

Reply:
https://github.com/votingworks/vxsuite/pull/4011/commits/9615ad30db98ac71f5561d4d4e9f20afe1b984e0 + https://github.com/votingworks/vxsuite/pull/4011/commits/4e2090f50673a9f8d7f99dee143e30f5d63999ac

This thread made me realize that there are quite a few operations that we need to block when a CVR sync is detected as necessary:

- Switching from official mode to test mode (assuming ballots have been counted)
- Deleting election data
- Closing polls (including from the polls paused state)

This video demonstrates all of the above:

https://github.com/votingworks/vxsuite/assets/12616928/7af274d9-0248-4784-9595-ee415a7635fd

While I was at it, I also had to tweak the flow for switching from official mode to test mode when ballots have been counted, which used to really explicitly bring the need to backup. I've now just swapped that modal out with a confirmation modal:

https://github.com/votingworks/vxsuite/assets/12616928/139d8bb4-bfc9-4a9e-9bcc-dc125337de4c